src/content/release-notes.mdx

```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.23.0
+
+12 August 2024
+
+This versions is compatible with Kubernetes versions 1.26 to 1.30
+
+### Deprecation Notice
+- Support for Kubernetes 1.26 has been deprecated and will be removed in the next release.
+
+### New Features
+
+- Added a [feature flag to enable/disable autowake](self-hosted/helm-configuration.mdx#autowake) for Namespaces and Preview Environments on incoming request  <!-- 8379 -->
+- Added support for Kubernetes 1.30  <!-- 8421 -->
+
+### Improvements
+
+- Added support for displaying the correct Dev Environment name when using Groups and Subgroups in GitLab
+- Fixed how Gitlab Groups and Subgroups are displayed in the Previews view
+- Prevent the usage of reserved variable names  <!-- 8272 -->
+- Updated registry version to return 404 errors for malformed image requests, allowing new builds of the image to be pushed. This resolves issues where improperly stored images prevented successful new builds  <!-- 8368 -->
+- Reconcile selected resource state consistently with URL  <!-- 8363 -->
+- Added a dashboard indicator to show whether the Garbage Collector is enabled  <!-- 8380 -->
+- Okteto CLI 2.30.0: You can now Build images using the `okteto build` command as part of your custom commands in the Okteto Manifest when executing a remote deploy/destroy. Previous to this, no image was being built
+- Okteto CLI 2.30.0: You can now [specify hosts](reference/okteto-manifest.mdx#hosts-string-optional) to resolve to specific IPs for your tests defined under the `test` section of the Okteto Manifest
+
+### Bug Fixes
+
+- Fix to gracefully handle localStorage errors  <!-- 8310 -->
+- Fixed layout shifting upon loading staged logs  <!-- 8406 -->
+- Fix "variables ignored" warning shown on deploy-URL for "Develop On Okteto Button" feature despite not having specified any variables  <!-- 8414 -->
+- Bumped daemon to 0.13.0 and cli to 2.30.0  <!-- 8411 -->
```

Comment:
I think this would be an improvement, right? Also, we usually link the CLI version to the release notes on GitHub for the CLI: https://github.com/okteto/okteto/releases/tag/2.30.0